packages/ui/src/1_atoms/Checkbox/Checkbox.module.css

```diff
@@ -5,8 +5,8 @@
 
 .input {
   @apply w-0 h-0 absolute;
-  top: -999999px;
-  left: -999999px;
+  top: 0;
+  left: 0;
```

Comment:
Why it was changed? Positioning was intentional to drag input out of the screen. Some browsers may show 2 checkboxes now (default input and custom one), because custom width/height for input checkbox is ignored